upup/pkg/fi/cloudup/new_cluster.go

```diff
@@ -769,6 +769,10 @@ func setupNetworking(opt *NewClusterOptions, cluster *api.Cluster) error {
 		cluster.Spec.Networking.Calico = &api.CalicoNetworkingSpec{
 			MajorVersion: "v3",
 		}
+		if cluster.Spec.KubeProxy == nil {
+			cluster.Spec.KubeProxy = &api.KubeProxyConfig{}
+		}
+		cluster.Spec.KubeProxy.Enabled = fi.Bool(false)
```

Comment:
I assume this is just for e2e testing

Reply:
Yeah. Checking failures, which are many.